File: dev-docs/snippets/group-type-configs/list.swift
import JsBaoClient

// List every group type configuration for the current app. Swift returns an
// untyped array of dictionaries rather than the typed `GroupTypeConfigInfo[]`.
func list(client: JsBaoClient) async throws {
  // #region example
  let configs = try await client.groupTypeConfigs.list()
  // #endregion example
  _ = configs
}

File: dev-docs/snippets/group-type-configs/list.ts
import type { JsBaoClient } from "js-bao-wss-client";

// List every group type configuration for the current app.
export async function list(client: JsBaoClient) {
  // #region example
  const configs = await client.groupTypeConfigs.list();
  // #endregion example
  return configs;
}
